src/table/table-role/grid-navigation.tsx

```diff
@@ -71,8 +72,7 @@ class GridNavigationProcessor {
     this.table.addEventListener('focusout', this.onFocusout);
     this.table.addEventListener('keydown', this.onKeydown);
 
-    muteElementFocusables(this.table, false);
```

Comment:
Not entirely sure what what this does but in the cleanup function we call `restoreElementFocusables`. Since we're no longer muting them, does it still need to be cleaned up?